onnxruntime/hosting/http/context.h

```diff
@@ -1,10 +1,20 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#ifndef ONNXRUNTIME_HOSTING_HTTP_HTTP_CONTEXT_H
-#define ONNXRUNTIME_HOSTING_HTTP_HTTP_CONTEXT_H
+#ifndef ONNXRUNTIME_HOSTING_HTTP_CONTEXT_H
+#define ONNXRUNTIME_HOSTING_HTTP_CONTEXT_H
+
+// boost random is using a deprecated header in 1.69
+// See: https://github.com/boostorg/random/issues/49
+#define BOOST_PENDING_INTEGER_LOG2_HPP
+#include <boost/integer/integer_log2.hpp>
+
+#include <string>
 
 #include <boost/beast/http.hpp>
```

Comment:
Another option would be libuuiddev, which need the `sudo apt-get install uuid-dev`

Here is an quick example: https://gist.github.com/yoggy/4483031

Reply:
On Windows, using `CoCreateGuid()`

Reply:
Leaving boost-uuid for now